cmd/cueckoo/cmd/cltrigger.go

```diff
@@ -188,6 +218,8 @@ func (c *cltrigger) triggerBuild(rev revision) error {
 		Fields: []string{"ALL_REVISIONS"},
 	})
 	if err != nil {
+		// Note that this may be a "change not found" error when the changeID is
```

Comment:
But doesn't this change mean that we will never be in this situation? Because if we are, Gerrit has bigger problems?

Reply:
You can still end up here if you run `cueckoo runtrybot --change Ideadbeef123` and that Change-Id string matches two CLs.

Reply:
Very good point, thanks.